lib/src/widgets/chat/ds_end_calls_message_bubble.widget.dart

```diff
@@ -0,0 +1,272 @@
+            controlForegroundColor: _isLightBubbleBackground
+                ? DSColors.neutralDarkRooftop
+                : DSColors.neutralLightSnow,
+            labelColor: _isLightBubbleBackground
+                ? DSColors.neutralDarkCity
+                : DSColors.neutralLightSnow,
+            bufferActiveTrackColor: _isLightBubbleBackground
+                ? DSColors.neutralMediumWave
+                : DSColors.neutralMediumElephant,
+            bufferInactiveTrackColor: _isLightBubbleBackground
+                ? DSColors.neutralDarkRooftop
+                : DSColors.neutralLightBox,
+            sliderActiveTrackColor: _isLightBubbleBackground
+                ? DSColors.primaryNight
+                : DSColors.primaryLight,
+            sliderThumbColor: _isLightBubbleBackground
+                ? DSColors.neutralDarkRooftop
+                : DSColors.neutralLightSnow,
+            speedForegroundColor: _isLightBubbleBackground
+                ? DSColors.neutralDarkCity
+                : DSColors.neutralLightSnow,
+            speedBorderColor: _isLightBubbleBackground
+                ? _isDefaultBubbleColors
+                    ? DSColors.neutralMediumSilver
+                    : DSColors.neutralDarkCity
+                : _isDefaultBubbleColors
+                    ? DSColors.disabledText
+                    : DSColors.neutralLightSnow,
+          ),
+        ),
+      );
+
+  Widget _buildLoading() => _buildContainer(
+        Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Row(
+            children: [
+              const DSSpinnerLoading(),
+              Padding(
+                padding: const EdgeInsets.only(left: 8.0),
+                child: DSCaptionText(
+                  'calls.preparing-record'.translate(),
+                  color: _foregroundColor,
+                ),
+              ),
+            ],
+          ),
+        ),
+      );
+
+  Widget _buildError() => _buildContainer(
+        Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              DSCaptionText(
+                'calls.load-record'.translate(),
+                color: _foregroundColor,
+              ),
+              DSButton(
+                onPressed: () => _streamController.add(true),
+                borderColor: _isLightBubbleBackground
+                    ? DSColors.neutralMediumSilver
+                    : DSColors.disabledText,
+                foregroundColor: _isLightBubbleBackground
+                    ? DSColors.neutralDarkCity
+                    : DSColors.neutralLightSnow,
+                backgroundColor: _isLightBubbleBackground
+                    ? DSColors.neutralLightWhisper
+                    : DSColors.neutralDarkDesk,
+                trailingIcon: const Icon(
+                  DSIcons.refresh_outline,
+                  size: 25.0,
```

Comment:
According to prototype, this must have a width of `24.0`.

Reply:
Done!